src/OpenTelemetry.Instrumentation.Dependencies/Implementation/SqlClientDiagnosticListener.cs

```diff
@@ -91,11 +90,10 @@ public override void OnCustom(string name, Activity activity, object payload)
 
                             if (this.commandTypeFetcher.Fetch(command) is CommandType commandType)
                             {
-                                activity.AddTag(DatabaseStatementTypeSpanAttributeKey, commandType.ToString());
-
                                 switch (commandType)
                                 {
                                     case CommandType.StoredProcedure:
+                                        activity.AddTag(DatabaseStatementTypeSpanAttributeKey, "StoredProcedure");
```

Comment:
Instead of using the magic string "StoredProcedure", i would use nameof(commandtype) or nameof(CommandType.StoredProcedure). The same comment applys for "Text"

Reply:
https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/trace/semantic_conventions/database.md Nothing here about "Type"! should we raise an issue to add it to spec?

Reply:
@eddynaka What you have against magic? j/k I added the `nameof` part in both Sql instrumentation classes.

@cijothomas I'll open an issue in spec for `db.statement_type` but my success rate there so far has been 0 😭 Maybe we should name it `db.mssql.statement_type`? Looking at the guide, what's interesting is that a lot of the database semantics have changed since I last worked on this instrumentation. I updated them on this PR:
* `db.type` is now `db.system` and we send `mssql` instead of `sql`.
* `db.instance` is now `db.name`.